src/components/Dashboard/Dashboard.jsx

```diff
@@ -130,35 +152,38 @@ const Dashboard = () => {
 				<div className="m-center mt-16 mb-16 w-3/4">
 					<div className="mb-10 flex items-center justify-between">
 						<Title>Bladwijzers</Title>
-						<button
-							onClick={() => setDeleteBookmark(!deleteBookmark)}
-							className={`h-10 rounded-lg  px-4 text-white ${deleteBookmark ? 'bg-slate-500' : 'bg-red-500'}`}
-						>
-							{deleteBookmark ? 'Annuleer' : 'Verwijder'}
-						</button>
+						{bookmarks.length > 0 && (
+							<button
+								id="delete"
+								onClick={() => setShowDeleteBookmark(!showDeleteBookmark)}
+								className={`h-10 rounded-lg  px-4 text-white ${showDeleteBookmark ? 'bg-slate-500' : 'bg-red-500'}`}
+							>
+								{showDeleteBookmark ? 'Annuleer' : 'Verwijder'}
+							</button>
+						)}
 					</div>
-					<div className="flex cursor-pointer flex-row flex-wrap justify-center ">
-						{bookmarks.map((bookmark) => (
+					<div className="grid grid-cols-5 gap-4">
+						{bookmarks.map(({ title, favicon, url }, index) => (
 							<BookmarkTile
-								key={bookmark.title}
-								title={bookmark.title}
-								img={bookmark?.favicon}
-								url={bookmark.url}
-								deleteBookmark={deleteBookmark}
+								ref={ref}
```

Comment:
Je maakt 1 ref aan en geeft die door aan meerdere Tiles... Kan je dit even uitleggen?

Reply:
Die ref zal gaan checken of je niet op een verwijder tegel klikt. Dit doet hij via de volgende functie: 
```
	const handleClickOutside = (e) => {
		if (ref.current && !ref.current.contains(e.target) && e.target.id !== 'delete') {
			setShowDeleteBookmark(false);
		}
	};
```

Indien je niet op een verwijder tegel klikt (dus ergens anders in het document), zal de showDeleteBookmark terug false worden

De ref wordt geforwarded omdat je niet rechtstreeks op een component een ref kan plaatsen.